config/database.yml

```diff
@@ -26,4 +26,4 @@ production:
   password: 
 
 cucumber:
-  <<: *test
+<<: *test
```

Comment:
@karenfreemansmith This shouldn't be changed. This property should be indented. Can you please revert back to its original state